pkg/controller.v1/common/pod.go

```diff
@@ -421,7 +421,7 @@ func (jc *JobController) createNewPod(job interface{}, rt string, index int, spe
 
 	// if gang-scheduling is enabled:
 	// 1. if user has specified other scheduler, we report a warning without overriding any fields.
-	// 2. if no SchedulerName is set for pods, then we set the SchedulerName to "volcano".
+	// 2. if no SchedulerName is set for pods, then we set the SchedulerName to enabled scheduler name.
```

Comment:
was this assuming that the default scheduler was volcano? Or is the comment innacurate?

Reply:
> was this assuming that the default scheduler was volcano?

We used to decorate the schedulerName in podTemplateSpec.spec.schedulerName only when the gang-scheduler name is set as `volcano`.

- volcano

https://github.com/kubeflow/common/blob/4dbf18c85417b54158914d1d01c1744b4e3542f2/pkg/controller.v1/control/podgroup_control.go#L65-L74

- scheduler-plugins

https://github.com/kubeflow/common/blob/4dbf18c85417b54158914d1d01c1744b4e3542f2/pkg/controller.v1/control/podgroup_control.go#L131-L136

Reply:
so the comment was inaccurate, right?

Reply:
I update the comment.